extensions/dicom-seg/src/components/BrushRadius/BrushRadius.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Range } from '@ohif/ui';
+
+import './BrushRadius.css';
+
+const BrushRadius = ({ value, onChange, min, max, step }) => (
+  <div className="dcmseg-brush-radius">
+    <label htmlFor="dcmseg-brush-radius">Brush Radius</label>
+    <Range
+      value={value}
+      min={min}
+      max={max}
+      step={step}
+      onChange={onChange}
+      id="dcmseg-brush-radius"
+    />
+  </div>
+);
+
+BrushRadius.propTypes = {
+  value: PropTypes.number,
+  onChange: PropTypes.number,
+  min: PropTypes.number,
+  max: PropTypes.number,
+  step: PropTypes.number,
+};
+
+BrushRadius.defaultProps = {
+  value: PropTypes.number.isRequired,
```

Comment:
Can you use this in `defaultProps`? I think you just wouldn't set the key here, and say it's required in propTypes?

Reply:
Fixed. thanks.